annotation/tests/test_annotations.py

```diff
@@ -50,6 +73,72 @@ class AnnotationRowTable(Base):
 )
 
 
+ANNOTATION_FILE_1 = File(
+    **{
+        "file_id": 1,
+        "tenant": TEST_TENANT,
+        "job_id": 1,
+        "pages_number": 10,
+    }
+)
+
+ANNOTATION_ANNOTATOR = User(user_id="6ffab2dd-3605-46d4-98a1-2d20011e132d")
+
+ANNOTATION_JOB_1 = Job(
+    **{
+        "job_id": 1,
+        "callback_url": "http://www.test.com/test1",
+        "annotators": [ANNOTATION_ANNOTATOR],
+        "validation_type": ValidationSchema.cross,
+        "files": [ANNOTATION_FILE_1],
+        "is_auto_distribution": False,
+        "categories": CATEGORIES,
+        "deadline": None,
+        "tenant": TEST_TENANT,
+        "job_type": JobTypeEnumSchema.ExtractionWithAnnotationJob,
```

Comment:
Same as in previous PR: make them fixtures, please

Reply:
Done